packages/docs/docs/dev/reference.mdx

```diff
@@ -271,3 +271,163 @@ This event is triggered when the Connectors list is updated. The callback functi
   file="../../examples/Connectors.tsx"
   commentBlock="eventConnectors"
 />
+
+## React Hooks
```

Comment:
I think it's better to create a new file, react hooks, to not mix the API of the fuel SDK with the hooks lib.

Reply:
I agree on this. 

cc @sarahschwartz